### PR TITLE
[Dev] use absolute paths for module imports ("../../../" fix)

### PR DESCRIPTION
**Pull request by l2succes**
Currently, we have many imports in the project with long and complex relative paths. This is not ideal as it makes it difficult to understand where imports are coming from. It's also painful to move files within the directory structure. To fix this issue, I converted a lot of our imports to use an absolute path where `lib` is the root.

So instead of
```javascript
import fonts from "../../../../../data/fonts"
```
we can write
```javascript
import fonts from "lib/data/fonts"
```

Absolute paths are supported by adding a `package.json`containing `{"name": "lib"}` inside of the `lib` folder. That way, node's module resolver will consider the folder as a package. For reference, I found this solution [here](https://github.com/facebook/react-native/issues/3099#issuecomment-221815006)

We could technically accomplish this using a combination of [typescript's path mapping](https://www.typescriptlang.org/docs/handbook/module-resolution.html) and [babel-plugin-module-resolver](https://github.com/tleunen/babel-plugin-module-resolver) but the `package.json` trick maintains VS Code autocomplete support for imports which makes adding imports way faster.

I also converted React imports from
```typescript
import * as React from "react"
```
to
```javascript
import React from "react"
```

This fixes the prop-types warning, you can read more about it [here](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/5128)

PS: @damassi We can do the same in reaction as well
Skip New Tests

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  No CHANGELOG added.
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/Artist/Articles/__tests__/index-tests.tsx`
- `src/lib/Components/Artist/Artworks/__tests__/index-tests.tsx`
- `src/lib/Components/Artist/Shows/__tests__/index-tests.tsx`
- `src/lib/Components/ArtworkGrids/__tests__/GenericGrid-tests.tsx`
- `src/lib/Components/Buttons/__tests__/Button-tests.tsx`
- `src/lib/Components/Buttons/__tests__/FlatWhite-tests.tsx`
- `src/lib/Components/Buttons/__tests__/index-tests.tsx`
- `src/lib/Components/Consignments/Components/__tests__/CloseButton-tests.tsx`
- `src/lib/Components/Consignments/Components/__tests__/DoneButton-tests.tsx`
- `src/lib/Components/Consignments/Components/__tests__/NavButton-tests.tsx`
- `src/lib/Components/Consignments/Typography/__tests__/index-tests.tsx`
- `src/lib/Components/__tests__/DottedLine-tests.tsx`
- `src/lib/Components/Home/ArtistRails/__tests__/ArtistRail-tests.tsx`
- `src/lib/Components/Home/ArtworkRails/__tests__/ArtworkRailHeader-tests.tsx`
- `src/lib/Components/Home/__tests__/HeroUnits-tests.tsx`
- `src/lib/Components/Home/__tests__/SearchBar-tests.tsx`
- `src/lib/Components/Home/__tests__/SectionTitle-tests.tsx`
- `src/lib/Components/Inbox/ActiveBids/__tests__/index-tests.tsx`
- `src/lib/Components/Inbox/Conversations/Preview/Attachment/__tests__/AttachmentPreview-tests.tsx`
- `src/lib/Components/Inbox/Conversations/__tests__/index-tests.tsx`
- `src/lib/Components/Inbox/Typography/__tests__/index-tests.tsx`
- `src/lib/Components/__tests__/LoadFailureView-tests.tsx`
- `src/lib/Components/__tests__/OpaqueImageView-tests.tsx`
- `src/lib/Components/RelatedArtists/__tests__/RelatedArtist-tests.tsx`
- `src/lib/Components/RelatedArtists/__tests__/index-tests.tsx`
- `src/lib/Components/__tests__/Spinner-tests.tsx`
- `src/lib/Components/__tests__/SwitchView-tests.tsx`
- `src/lib/Components/Text/__tests__/Headline-tests.tsx`
- `src/lib/Components/Text/__tests__/Serif-tests.tsx`
- `src/lib/Scenes/Home/Components/ForYou/Components/__tests__/ArtworkCarousel-tests.tsx`
- `src/lib/Scenes/Home/Components/ForYou/Components/__tests__/ArtworkCarouselHeader-tests.tsx`
- `src/lib/Scenes/Home/Components/ForYou/__tests__/index-tests.tsx`
- `src/lib/Scenes/Home/Components/Sales/Relay/__tests__/SalesRenderer-tests.tsx`
- `src/lib/Scenes/Home/Components/__tests__/SectionTitle-tests.tsx`
- `src/lib/data/__tests__/colors-tests.ts`
- `src/lib/data/__tests__/fonts-tests.ts`
- `src/lib/utils/__tests__/renderWithLoadProgress-tests.tsx`
- `src/lib/utils/__tests__/track-tests.ts`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/Artist/Articles/__tests__/index-tests.tsx`
- `src/lib/Components/Artist/Artworks/__tests__/index-tests.tsx`
- `src/lib/Components/Artist/Shows/__tests__/index-tests.tsx`
- `src/lib/Components/ArtworkGrids/__tests__/GenericGrid-tests.tsx`
- `src/lib/Components/Buttons/__tests__/Button-tests.tsx`
- `src/lib/Components/Buttons/__tests__/FlatWhite-tests.tsx`
- `src/lib/Components/Buttons/__tests__/index-tests.tsx`
- `src/lib/Components/Consignments/Components/__tests__/CloseButton-tests.tsx`
- `src/lib/Components/Consignments/Components/__tests__/DoneButton-tests.tsx`
- `src/lib/Components/Consignments/Components/__tests__/NavButton-tests.tsx`
- `src/lib/Components/Consignments/Typography/__tests__/index-tests.tsx`
- `src/lib/Components/__tests__/DottedLine-tests.tsx`
- `src/lib/Components/Home/ArtistRails/__tests__/ArtistRail-tests.tsx`
- `src/lib/Components/Home/ArtworkRails/__tests__/ArtworkRailHeader-tests.tsx`
- `src/lib/Components/Home/__tests__/HeroUnits-tests.tsx`
- `src/lib/Components/Home/__tests__/SearchBar-tests.tsx`
- `src/lib/Components/Home/__tests__/SectionTitle-tests.tsx`
- `src/lib/Components/Inbox/ActiveBids/__tests__/index-tests.tsx`
- `src/lib/Components/Inbox/Conversations/Preview/Attachment/__tests__/AttachmentPreview-tests.tsx`
- `src/lib/Components/Inbox/Conversations/__tests__/index-tests.tsx`
- `src/lib/Components/Inbox/Typography/__tests__/index-tests.tsx`
- `src/lib/Components/__tests__/LoadFailureView-tests.tsx`
- `src/lib/Components/__tests__/OpaqueImageView-tests.tsx`
- `src/lib/Components/RelatedArtists/__tests__/RelatedArtist-tests.tsx`
- `src/lib/Components/RelatedArtists/__tests__/index-tests.tsx`
- `src/lib/Components/__tests__/Spinner-tests.tsx`
- `src/lib/Components/__tests__/SwitchView-tests.tsx`
- `src/lib/Components/Text/__tests__/Headline-tests.tsx`
- `src/lib/Components/Text/__tests__/Serif-tests.tsx`
- `src/lib/Scenes/Home/Components/ForYou/Components/__tests__/ArtworkCarousel-tests.tsx`
- `src/lib/Scenes/Home/Components/ForYou/Components/__tests__/ArtworkCarouselHeader-tests.tsx`
- `src/lib/Scenes/Home/Components/ForYou/__tests__/index-tests.tsx`
- `src/lib/Scenes/Home/Components/Sales/Relay/__tests__/SalesRenderer-tests.tsx`
- `src/lib/Scenes/Home/Components/__tests__/SectionTitle-tests.tsx`
- `src/lib/data/__tests__/colors-tests.ts`
- `src/lib/data/__tests__/fonts-tests.ts`
- `src/lib/utils/__tests__/renderWithLoadProgress-tests.tsx`
- `src/lib/utils/__tests__/track-tests.ts`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
 ###### Dev
 
+-   Converts relative to absolute paths for imports - luc
```
Comment (orta) [inline CHANGELOG.md]: We shipped this beta already - add your changelog notes inside  the master section for the next release 👍

File: src/lib/Components/Artist/Articles/Article.tsx [hunk 2]
```diff
@@ -1,10 +1,10 @@
-import * as React from "react"
+import React from "react"
```
Comment (ashfurrow) [inline src/lib/Components/Artist/Articles/Article.tsx]: This looks nicer to me – what's the difference? Is there any rationale to keep the old way?

Comment (ashfurrow) [reply src/lib/Components/Artist/Articles/Article.tsx]: Argh, just re-read your PR description, got it 👍

**Comment by orta** [comment]
OK, let's do this.

**Comment by orta** [comment]
OK, let's do this.

File: src/lib/Components/Artist/Articles/Article.tsx (hunk 2 above)
Comment (l2succes) [reply src/lib/Components/Artist/Articles/Article.tsx]: :)

**Pull request merged**